### ISSUE TITLE
Ie 9, Object doesn't support property or method 'setCustomValidity'

### ISSUE THREAD

**Issue by andalt** [telerik/kendo-react#105]
In ie9 do not work components Input, DropDownList, DatePicker and others.

Error: Object doesn't support property or method 'setCustomValidity'

```
import 'core-js/es6/map';
import 'core-js/es6/set';
import 'core-js/shim';
import 'raf/polyfill';

import React from 'react';
import { Input } from '@progress/kendo-react-inputs';

const Search = () => (
    <div className="row">
        <div className="col-xs-6">
            <Input />
        </div>
    </div>
);

export default Search;
```

--------------------------------------

"react": "^16.5.2",
"@progress/kendo-react-inputs": "^2.1.0",

![2018-10-22_17 23 05_ko7im](https://user-images.githubusercontent.com/8655528/47297960-41bc7480-d61f-11e8-8785-c4501a4f6d98.png)

**Comment by Xizario** [comment]
Fixed in `v2.3.1-dev.201810291338`

**Comment by andalt** [comment]
New Bug

"@progress/kendo-react-inputs": "^2.3.1",

![2018-11-01_14 49 35_o7iqb](https://user-images.githubusercontent.com/8655528/47850193-84cbd400-dde5-11e8-8e18-fcd213dcf4f6.png)
